oat-sa/tao-core
# Fix/aut 2349/translate rdf properties

Related to:
https://oat-sa.atlassian.net/browse/AUT-2349

**PR CONTAINS:**
- Fix for rdf labels.

- update of Translation files with latest words.

- Addition of missing translations as required by ticket (TestTaker tab)

STEPS TO TEST:

Go to the `testTakers` tab and check that all elements are translated for :
`lt-LT` Lithuanian
`nl-NL` Dutch
`de-DE` German
`ja-JP ` Japan

CURRENT RESULT:
Some of the elements in the form were not translated

EXPECTED RESULT:

All elements are translated for all the above languages as image below

![image](https://user-images.githubusercontent.com/60346520/203806065-b2345779-4385-47eb-91a0-0ac0d8b2765a.png)
